### ISSUE TITLE
Sorting and Grouping for Variable Templates

### ISSUE THREAD

**Issue by michaelnoonan** [OctopusDeploy/Issues#2733]
In Octopus 3.4.0 we display the Variable Templates in the order they are created. At the very least we should allow people to reorder their Variable Templates so they can control their Tenant Variable pages, which would allow for some rudimentary grouping. In addition it would be really useful for people to specify a group for each Variable Template so we can render the forms as groups of variables with group headers.

The linked source provides a good example of where allowing people to provide logical groupings of Variable Templates would be really useful.

Source: http://help.octopusdeploy.com/discussions/beta-testing-feedback/133-project-template-variable-sets-34-beta-feedback

**Comment by michaelnoonan** [comment]
I would like to see this kind of thing happen too, but it hasn't come to the top of the "priority-vs-complexity" list. At this point it we think it will be implemented as a first-class citizen in remote release promotions: https://octopus.com/blog/remote-release-promotions-rfc

**Comment by ruant** [comment]
Yes! Please add this feature. Would be so much easier to work with.
